bin/testrepl.py

```diff
@@ -1,25 +0,0 @@
-import click
-from click_repl import register_repl
-
```

Comment:
why should we remove this file?

Reply:
that test case is not at all being involved
I thought of adding that test in the next pr

If u want me to add that test, I can make an another commit for that